plugins/vsphere/pkg/command/binding.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2020 VMware, Inc.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package command
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/spf13/cobra"
+	"github.com/vmware-tanzu/sources-for-knative/pkg/apis/sources/v1alpha1"
+	"github.com/vmware-tanzu/sources-for-knative/plugins/vsphere/pkg"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/apis"
+	duckv1alpha1 "knative.dev/pkg/apis/duck/v1alpha1"
+	"knative.dev/pkg/tracker"
+)
+
+type BindingOptions struct {
+	Namespace     string
+	Name          string
+	Address       string
+	SkipTlsVerify bool
+	SecretRef     string
```

Comment:
Both commands have these, and we should ensure they are consistently surfaced.  Perhaps a base VAuthOptions?  OK to follow up.